source/install_cli.rst

```diff
@@ -2,39 +2,15 @@
 
 Installing the Quipucords Command Line Tool
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-qpc, the command line tool that is installed by RPM, is available for `download <https://copr.fedorainfracloud.org/coprs/g/quipucords/qpc/>`_ from the Fedora COPR build and repository system. Use the following steps to install the command line tool.
-
-Download and Configure EPEL
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-1. Enable the EPEL repo for the server. You can find the appropriate architecture and version on the `EPEL wiki <https://fedoraproject.org/wiki/EPEL>`_.
-
-  - For Red Hat Enterprise Linux 7 or CentOS 7, enter the following command::
-
-      # rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-
-  - For Red Hat Enterprise Linux 6 or CentOS 6, enter the following command::
-
-      # rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
-
-2. Add the COPR repo to your server. You can find the appropriate architecture and version on the `COPR qpc page <https://copr.fedorainfracloud.org/coprs/g/quipucords/qpc/>`_.
-
-
-  - For Red Hat Enterprise Linux 7 or CentOS 7, enter the following command::
-
-      # wget -O /etc/yum.repos.d/group_quipucords-qpc-epel-7.repo https://copr.fedorainfracloud.org/coprs/g/quipucords/qpc/repo/epel-7/group_quipucords-qpc-epel-7.repo
-
-  - For Red Hat Enterprise Linux 6 or CentOS 6, enter the following command::
-
-      # wget -O /etc/yum.repos.d/group_quipucords-qpc-epel-6.repo https://copr.fedorainfracloud.org/coprs/g/quipucords/qpc/repo/epel-6/group_quipucords-qpc-epel-6.repo
-
+qpc, the command line tool that is installed by RPM, is available for `download <https://github.com/quipucords/qpc/releases/latest/>`_ from the GitHub. Use the following steps to install the command line tool.
```

Comment:
I don't think that the `the` is needed here. `from the GitHub` => `from GitHub`

Reply:
Now you are going to tell me it's not the Facebook